spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -273,6 +273,7 @@
 "GWT Terms": Apache-2.0 AND BSD-3-Clause AND CC0-1.0 AND EPL-1.0 AND LGPL-2.1-only AND MPL-1.1
 "General Public License (GPL)": GPL-2.0-only
 "General Public License 2.0 (GPL)": GPL-2.0-only
+"Google Maps Platform Terms of Service": LicenseRef-ort-google-maps-tos
```

Comment:
Can you please split this change, as it's less controversial than the other change, out to another PR, or drop the other change from this PR?

Reply:
Done: https://github.com/oss-review-toolkit/ort/pull/3337